libutils/string_lib.c

```diff
@@ -1801,3 +1801,27 @@ char *StringJoin(const Seq *const seq, const char *sep)
     char *const data = StringWriterClose(writer);
     return data;
 }
+
+Seq *StringSplit(const char *const str, const char *const del)
```

Comment:
We already have `SeqStringFromString()`.

Reply:
`SeqStringFromString()` only splits on **one** delimiter

Reply:
You're right. But then those two functions should be next to each other perhaps with a comment pointing out the difference.

Reply:
Added comment about similarities. However, I feel like `StringSplit()` belongs next to `StringJoin()`

Reply:
That is indeed something to worry about. However, `StringJoin()` already uses things from `sequence.h`.

Reply:
@larsewi `SeqStringFromString()` should probably be moved to this file, and renamed to something more descriptive like `StringSplitSingleChar()`. Then all the string split and joins are in the same place :)

Reply:
Would not renaming it would cause a braking change to the libntech API?

Reply:
Yes, it would be. It could have two names for a period of time, though. A bigger concern for me is the `#include` circular dependency.